server/app.ts

```diff
@@ -96,6 +96,12 @@ export default function createApp(
             "'sha256-pu37IY9Ig/H/vwy6rKyd1hbGd0HCmI3jhdUIhUV1l2A='",
             "'sha256-v8tCeJtLZcGYwtw7d1C1rXv3Tv2Clp+NyDAzPWZvtaY='",
             "'sha256-MoTaAoB/V8e6T3/hagtzBDVnvuUwgTboZeP40OVsw4k='",
+            "'sha256-5jMuXDZya0nKx/E7KeO2htVzsDvBbBGHk54V7ZEFCwM='",
```

Comment:
We probably need to make sure we are removing the ones that are no longer used when updating old scripts. Not necessarily now, but this list will keep growing otherwise, where a bunch are redundant.

Reply:
@rgforsyth I agree to your point. probably after this goes in, we can take up that exercise.

Reply:
👍